pwiz_tools/Skyline/TestTutorial/GroupedStudiesTutorialTest.cs

```diff
@@ -127,7 +125,7 @@ protected override void DoTest()
                 RunUI(() =>
                 {
                         editIrtCalcDlg.CalcName = irtCalcName;
-                        editIrtCalcDlg.CreateDatabase(GetOcRawTestPath("iRT-OC-Study.irtdb")); // Not L10N
+                        editIrtCalcDlg.CreateDatabase(GetTestPath("iRT-OC-Study.irtdb")); // Not L10N

```

Comment:
Okay. Though, in the true sense of the original we would want the .sky file to at least go into the "Ovarian Cancer" folder, because it was meant to share space with the "Heart Failure" study in GroupedStudies1TutorialTest. This probably should have been GroupedStudies2TutorialTest.